packages/react-components/src/organisms/Filter.tsx

```diff
@@ -54,26 +55,23 @@ const showMenuStyles = css({
 interface FilterProps<V extends string> {
   readonly filters?: Set<V>;
   readonly onChangeFilter?: (filter: V) => void;
-  readonly filterOptions: Option<V>[];
-  readonly filterTitle: string;
+  readonly filterOptions: ReadonlyArray<Option<V> | Title>;
 }
 export default function Filter<V extends string>({
   filters = new Set(),
   onChangeFilter = noop,
   filterOptions,
-  filterTitle,
 }: FilterProps<V>): ReturnType<React.FC> {
   const [menuShown, setMenuShown] = useState(false);
   useEffect(() => {
     setMenuShown(false);
-  }, [filterTitle]);
+  }, [filterOptions]);
 
   const [debouncedFilters] = useDebounce(filters, 5000);
   useEffect(() => {
     if (filters.size && filters === debouncedFilters) {
       window.dataLayer?.push({
         [FILTERS_KEY]: [...filters],
```

Comment:
This is for analytics. Not sure if we should drop this or need to do something else